module4/pd/lessons/tech_pathways.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+title: Tech Pathways
+module: 4
+---
+
+## Expanding Your Career Horizons
+
+The journey into tech doesn’t have just one destination—there are countless career paths beyond traditional software engineering! Tech Pathways is a four-part lesson series designed to introduce you to the wide range of roles available in the industry. Whether you're passionate about coding, data, writing, community, etc. there’s a place for you in tech. _You_ belong here! These lessons will break down both technical and non-technical roles, helping you explore career options that align with your unique skills and interests. From Site Reliability Engineering to Developer Relations, Product Management to Solutions Engineering, you’ll gain insight into how different roles contribute to building, supporting, and scaling technology.
```

Comment:
I really love the updates to this paragraph. It definitely amps up the excitement for me.

Reply:
Woo! Im glad you think so!